components/textfield/themes/spectrum.css

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@container (--system: spectrum) {
+  .spectrum-Textfield {
+    --spectrum-textfield-border-color: var(--spectrum-gray-500);
+    --spectrum-textfield-border-color-hover: var(--spectrum-gray-600);
+    --spectrum-textfield-border-color-focus: var(--spectrum-gray-800);
+    --spectrum-textfield-border-color-focus-hover: var(--spectrum-gray-900);
+    --spectrum-textfield-border-color-keyboard-focus: var(--spectrum-gray-900);
+
+    --spectrum-textfield-border-width: var(--spectrum-border-width-100);
```

Comment:
These custom properties appear to be the same both here and in `express`. I think we should try to move these into the `index.css` file, unless I'm missing some context as to why they'd need to be here and in `express.css`

Reply:
I'm so glad you noticed this, the token for Express should have been --spectrum-border-width-200. Fixed.